From fb7d198b5687299606a76fffd2485a9b3dc92ab2 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Fri, 26 Oct 2012 16:53:05 +0800
Subject: [PATCH 088/310] ENGR00230369 imx6 sabresd: refine the keyguard lock
 screen layout

When the system font size is set to quite large,
the digital time widget can not show the full time
as the keyguard screen status land layout width
is too small. Here just decrease the layout left and
right margin to display the time correctly.

The actual differences is like:
@@ -42,9 +42,9 @@
<include layout="@layout/keyguard_screen_status_land"
		 android:layout_width="match_parent"
		 android:layout_height="wrap_content"
-                android:layout_marginLeft="50dip"
+                android:layout_marginLeft="20dip"
		 android:layout_marginTop="50dip"
-                android:layout_marginRight="64dip"
+                android:layout_marginRight="30dip"
		 android:layout_alignParentTop="true"
		 android:layout_alignParentLeft="true"/>

Change-Id: I39c9c1f8dcfc4aee331a60308ba55efc3c0f770a
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 .../keyguard_screen_tab_unlock_land.xml            | 124 +++++++++++++++++++++
 1 file changed, 124 insertions(+)
 create mode 100644 imx6/sabresd/overlay/frameworks/base/core/res/res/layout-sw600dp/keyguard_screen_tab_unlock_land.xml

diff --git a/imx6/sabresd/overlay/frameworks/base/core/res/res/layout-sw600dp/keyguard_screen_tab_unlock_land.xml b/imx6/sabresd/overlay/frameworks/base/core/res/res/layout-sw600dp/keyguard_screen_tab_unlock_land.xml
new file mode 100644
index 0000000..8058ee8
--- /dev/null
+++ b/imx6/sabresd/overlay/frameworks/base/core/res/res/layout-sw600dp/keyguard_screen_tab_unlock_land.xml
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+**
+** Copyright 2009, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License")
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+
+<!-- This is the general lock screen which shows information about the
+  state of the device, as well as instructions on how to get past it
+  depending on the state of the device.-->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tabunlock="http://schemas.android.com/apk/res/com.android.tabunlock"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="horizontal"
+    android:id="@+id/root">
+
+    <!-- left side: status and music -->
+    <RelativeLayout
+        android:layout_height="match_parent"
+        android:layout_weight="0.42"
+        android:layout_width="0dip"
+        android:gravity="center">
+
+        <RelativeLayout android:id="@+id/transport_bg_protect"
+            android:layout_width="512dip"
+            android:layout_height="wrap_content">
+
+            <!-- Status -->
+            <include layout="@layout/keyguard_screen_status_land"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="20dip"
+                android:layout_marginTop="50dip"
+                android:layout_marginRight="30dip"
+                android:layout_alignParentTop="true"
+                android:layout_alignParentLeft="true"/>
+
+            <!-- Music transport control -->
+            <include android:id="@+id/transport"
+                layout="@layout/keyguard_transport_control"
+                android:layout_row="0"
+                android:layout_column="0"
+                android:layout_rowSpan="3"
+                android:layout_columnSpan="1"
+                android:layout_gravity="fill"
+                android:layout_width="match_parent"
+                android:layout_height="512dip"
+                />
+
+        </RelativeLayout>
+
+    </RelativeLayout>
+
+    <!-- right side -->
+    <RelativeLayout
+        android:layout_height="match_parent"
+        android:layout_weight="0.58"
+        android:layout_width="0dip"
+        android:gravity="center_horizontal|center_vertical">
+
+        <TextView
+            android:id="@+id/screenLocked"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textAppearance="?android:attr/textAppearanceMedium"
+            android:gravity="center"
+            android:layout_marginTop="12dip"
+            android:layout_alignParentLeft="true"
+            android:layout_alignParentTop="true"
+            android:drawablePadding="4dip"/>
+
+        <com.android.internal.widget.multiwaveview.GlowPadView
+            android:id="@+id/unlock_widget"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_rowSpan="7"
+            android:layout_gravity="center_vertical|right"
+            android:gravity="center"
+            android:focusable="true"
+
+            android:targetDrawables="@array/lockscreen_targets_with_camera"
+            android:targetDescriptions="@array/lockscreen_target_descriptions_with_camera"
+            android:directionDescriptions="@array/lockscreen_direction_descriptions"
+            android:handleDrawable="@drawable/ic_lockscreen_handle"
+            android:outerRingDrawable="@drawable/ic_lockscreen_outerring"
+            android:outerRadius="@dimen/glowpadview_target_placement_radius"
+            android:innerRadius="@dimen/glowpadview_inner_radius"
+            android:snapMargin="@dimen/glowpadview_snap_margin"
+            android:feedbackCount="1"
+            android:vibrationDuration="20"
+            android:glowRadius="@dimen/glowpadview_glow_radius"
+            android:pointDrawable="@drawable/ic_lockscreen_glowdot"
+        />
+
+        <!-- emergency call button shown when sim is PUKd and tab_selector is hidden -->
+        <Button
+            android:id="@+id/emergencyCallButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginRight="80dip"
+            android:layout_marginBottom="80dip"
+            android:layout_alignParentRight="true"
+            android:layout_alignParentBottom="true"
+            android:drawableLeft="@drawable/ic_emergency"
+            style="@style/Widget.Button.Transparent"
+            android:drawablePadding="8dip"
+            android:visibility="gone"/>
+
+    </RelativeLayout>>
+
+</LinearLayout>
-- 
1.8.0

